hub/models.py

```diff
@@ -1497,11 +1500,12 @@ async def import_page(self, page: int) -> bool:
 
     async def import_many(self, members: list):
         """
+        Members doesn't mean members, it's actually generic records.
```

Comment:
Let's rename `member` to `record` in a separate ticket :) https://linear.app/commonknowledge/issue/MAP-677/rename-member-to-record-in-modelspy